src/CommandLineParser.ts

```diff
@@ -16,6 +16,13 @@ export const argv: any = yargs.options({
         describe: 'The username used to log into Microsoft Stream (enabling this will fill in the email field for you).',
         demandOption: false
     },
+    concat: { // FOR CONCATENATING VIDEOS
+        alias: 'c',
+        describe: 'If enabled concatenate videos in the order they are typed in the file',
+        type: 'boolean',
+        default: false,
+        demandOption: false
+    },
```

Comment:
Please move it down the options list, below closedCaptions, so that the required/most important options can be seen first